### PR TITLE
Send announcements page

### PR DESCRIPTION
**Pull request by SirSaunders**
-  fixed posting announcements from the website 
-  added FCM notification sending from backend when announcements are posted
- FCM key is currently hard coded, may want that as an environmental var, we do not get charged for it so not super important.
- Also not sure if you we have a requerments.txt for python but I added` json` and `requests` packages

### CODE REVIEW

File: backend/Annoucements.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
 import ids
 import time
 import Slack
+import requests
+import json
 
 myfirebase = ids.myfirebase
+FCM_KEY = 'AAAAfPkLsFE:APA91bH7vQ6_InhI1tBrr_k3tFCm7w3SvlU8Ba0FyrXE7GtOJN7dZEKctdW9Ota8mneIDeJA5Ddambw-ugyjbMZPyV9QkPtNft9z2FtIQ7WG_Ql7OHgYgnZ3dc8f-z334rTR8wjNkKDi'
```
Comment (TylerCarberry) [inline backend/Annoucements.py]: Aaaahhhh! 😱Force push and remove this!

**Review by TylerCarberry — CHANGES_REQUESTED** [review]

File: backend/Annoucements.py (hunk 1 above)
Comment (SirSaunders) [reply backend/Annoucements.py]: Ill just hot fix it now 🙄 and change the key.

Comment (SirSaunders) [reply backend/Annoucements.py]: lol but now it is in the comments on github

**Comment by SirSaunders** [comment]
can we merge yet D:

File: backend/Annoucements.py (hunk 1 above)
Comment (TylerCarberry) [reply backend/Annoucements.py]: As long as you changed the api key it's fine

**Comment by TylerCarberry** [comment]
sure ¯\\\_(ツ)\_/¯

**Comment by SirSaunders** [comment]
D,:

**Comment by TylerCarberry** [comment]
I'm not the president anymore. You'd have to ask RJ or @AllTheMegahertz what he's planning on doing about the website and apps

**Comment by SirSaunders** [comment]
totally, closing this XD I can not stand with this being my only open pull request.

**Pull request closed without merging**